Tasks/NotationV0/task.json

```diff
@@ -0,0 +1,251 @@
+      },
+      "groupName": "pluginConfig"
+    },
+    {
+      "name": "akvPluginVersion",
+      "type": "string",
+      "label": "Plugin Version",
+      "defaultValue": "1.0.1",
+      "required": true,
+      "helpMarkDown": "The version of the Azure Key Vault plugin to be installed. please visit the [release page](https://github.com/Azure/notation-azure-kv/releases) for the available versions.",
+      "visibleRule": "plugin = azureKeyVault",
+      "groupName": "pluginConfig"
+    },
+    {
+      "name": "azurekvServiceConection",
+      "type": "connectedService:AzureRM",
+      "label": "Azure Key Vault service connection",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Select the Azure subscription for the key vault if prefer to use service connection for authentication.",
+      "visibleRule": "plugin = azureKeyVault",
+      "groupName": "pluginConfig"
+    },
+    {
+      "name": "keyid",
+      "type": "string",
+      "label": "Key ID",
+      "required": true,
+      "defaultValue": "",
+      "helpMarkDown": "The Key ID is the key or certificate identifier for Azure Key Vault.",
+      "visibleRule": "plugin = azureKeyVault",
+      "groupName": "pluginConfig"
+    },
+    {
+      "name": "caCertBundle",
+      "type": "string",
+      "label": "Certificate Bundle File Path",
+      "required": false,
+      "defaultValue": "",
+      "visibleRule": "plugin = azureKeyVault",
+      "helpMarkDown": "A file with root and all intermediate certificates, starting from the root certificate, following the order in the certificate chain.",
+      "groupName": "pluginConfig"
+    },
+    {
+      "name": "selfSigned",
+      "type": "boolean",
+      "label": "Self-signed Certificate",
+      "required": false,
+      "defaultValue": false,
+      "visibleRule": "plugin = azureKeyVault",
+      "groupName": "pluginConfig"
+    },
+    {
+      "name": "trustPolicy",
+      "type": "string",
+      "label": "Trust Policy File Path",
+      "required": true,
+      "defaultValue": "",
+      "visibleRule": "command = verify",
+      "helpMarkDown": "The path to the [trust policy](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md#trust-policy) file relative to the repository. Example: ./path/to/trust-policy.json",
+      "groupName": "commandConfig"
+    },
+    {
+      "name": "trustStore",
+      "type": "string",
+      "label": "Trust Store Folder Path",
+      "required": true,
+      "defaultValue": "",
+      "visibleRule": "command = verify",
+      "helpMarkDown": "The path to the directory containing the [trust store](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md#trust-store) relative to the repository. Example: ./path/to/truststore/",
+      "groupName": "commandConfig"
+    }
+  ],
+  "execution": {
+    "Node10": {
+      "target": "src/index.js",
+      "argumentFormat": ""
+    },
+    "Node16": {
```

Comment:
this should be replaced by node20 (and tested on node20)

Reply:
node16 is out of support by now.

Reply:
We can remove the node10 handler, and it's okay to keep node16 handler for now, as there's node16 runner on agent we can rollback to, if node20 runner not working. We can keep node16 and node20 under execution in task.json file.

Reply:
Updated to keep node 16 and node 20 for now.